web-ui/src/main/resources/catalog/templates/admin/settings/system.html

```diff
@@ -53,7 +53,7 @@
                 translate}}</label>
               <!-- According to entry -->
               <div class="col-sm-8" data-ng-switch="s['@name']">
-
+								{{s['@name]'}}
```

Comment:
It breaks the admin https://github.com/geonetwork/core-geonetwork/issues/856
and if displayed we get the setting key on top of each field 
![image](https://cloud.githubusercontent.com/assets/1701393/6613390/48311e00-c88c-11e4-8df2-4f866d2a42fe.png)